js/helper.js

```diff
@@ -15,6 +15,7 @@ function getBinaryPath() {
   const parts = [];
```

Comment:
Could we use `require.resolve` instead here, like this:
```javascript
path.join(require.resolve('@sentry/cli'), `../../bin/sentry-cli${process.platform === 'win32' ? '.exe' : ''}`)
```

Reply:
No, unfortunately, we specifically cannot use `require.resolve` due to `@vercel/nft` behavior.

See https://github.com/getsentry/sentry-cli/pull/1207 and a follow-up in https://github.com/getsentry/sentry-cli/pull/1374

> Perhaps could have something to do with the fact that it's a monorepo? I'm not entirely sure, but it's worth looking into.

Yes, that's most likely it, as it's making links between shared packages in the root directory. I'm out for the next few weeks, but will keep that issue in mind.